com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/GraphicsToolsStandard.shader

```diff
@@ -34,6 +34,7 @@ Shader "Graphics Tools/Standard"
 
         // Rendering options.
         [Enum(Microsoft.MixedReality.GraphicsTools.Editor.LightMode)] _DirectionalLight("Light Mode", Float) = 1.0 // "LitDirectional"
+        [Toggle(_NON_PHOTOREALISTIC)] _NPR("Non-Photorealistic Rendering", Float) = 0.0
```

Comment:
Could you also add this property to the `GraphicsToolsStandardCanvas` shader too? Without it you get this error in Unity:

> ArgumentException: Could not find MaterialProperty: '_NPR', Num properties: 117